pkgs/blast_repo/lib/src/tweaks/auto_publish_tweak.dart

```diff
@@ -55,4 +55,7 @@ jobs:
   publish:
     if: ${{ github.repository_owner == '{org}' }}
     uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main
+    permissions:
```

Comment:
Are these just needed (the explicit permission mentions) if consuming the workflow from a non `dart-lang` repo? I imagine it's good to be explicit in any case.

Reply:
maybe? Seems to be where I'm hitting it!